airdrop/infrastructure/repositories/airdrop_repository.py

```diff
@@ -165,7 +165,15 @@ def get_token_address(self, airdrop_id):
         if airdrop is None:
             raise Exception("Airdrop not found")
 
-        return airdrop.address
+        return airdrop.token_address
+
+    def get_staking_contract_address(self, airdrop_id):
```

Comment:
This is not the right way to query the database.

Reply:
Fixed in the latest commit